types/react-native/index.d.ts

```diff
@@ -3662,7 +3662,7 @@ export interface SectionListData<ItemT> extends SectionBase<ItemT> {
     [key: string]: any;
 }
 
-export interface SectionListProperties<ItemT> extends ScrollViewProperties {
+export interface SectionListProperties<ItemT> extends VirtualizedListProperties<ItemT> {
```

Comment:
SectionList props extends VirtualizedSectionListProps:
[0.52-stable](https://github.com/facebook/react-native/blob/0.52-stable/Libraries/Lists/SectionList.js#L192)
[0.53-stable](https://github.com/facebook/react-native/blob/0.53-stable/Libraries/Lists/SectionList.js#L192)
[0.54-stable](https://github.com/facebook/react-native/blob/0.53-stable/Libraries/Lists/SectionList.js#L192)
[0.55-stable](https://github.com/facebook/react-native/blob/0.55-stable/Libraries/Lists/SectionList.js#L190)

@tn0502 Can you modify this line so we preserve `interface SectionListProperties<ItemT> extends VirtualizedListProperties<ItemT>`?